propertyestimator/workflow/protocols.py

```diff
@@ -992,14 +1012,25 @@ def _setup_new_simulation(self, directory, temperature, pressure, available_reso
                                     integrator,
                                     platform)
 
-        box_vectors = input_pdb_file.topology.getPeriodicBoxVectors()
+        checkpoint_path = path.join(directory, 'trajectory.chk')
```

Comment:
I don't think the `CheckpointReporter` stores the full trajectory. I believe only the last snapshot is saved on disk (and overwrites the old one).

Reply:
Good catch - this file was meant to just be called something like `checkpoint.chk` but didn't get changed when copy pasting from above!